R/grid_search.R

```diff
@@ -40,7 +42,8 @@ scan_beta_date <- function(
   first_start_date, 
   last_start_date, 
   day_step,
-  data, 
+  data,
+  model = NULL,
```

Comment:
you can pass in the string `"new_hospital_model"` you get the new hospital model